code/modules/admin/admin.dm

```diff
@@ -624,43 +624,95 @@ GLOBAL_VAR_INIT(nologevent, 0)
 
 ////////////////////////////////////////////////////////////////////////////////////////////////ADMIN HELPER PROCS
 
-/proc/is_special_character(mob/M as mob) // returns 1 for specail characters and 2 for heroes of gamemode
+/**
+  * A proc that return whether the mob is a "Special Character" aka Antagonist
+  *
+  *
+  *
+  * Arguments:
+  * * M - the mob you're checking
+  * *
+  */
+/proc/is_special_character(mob/M as mob)
+	if(!SSticker || !SSticker.mode)
```

Comment:
```suggestion
/**
  * A proc that return whether the mob is a "Special Character" aka Antagonist
  *
  * Arguments:
  * * M - the mob you're checking
  */
/proc/is_special_character(mob/M as mob)
	if(!SSticker.mode)
```

Cleans up doc and we dont need any `!SSticker` checks. `SSticker` will always exist.